config/config.yml

```diff
@@ -473,3 +473,7 @@ PERMITTED_HOSTS: [
   "test.archiveofourown.org",
   "testdownload.archiveofourown.org"
 ]
+
+USERNAME_CHANGE:
```

Comment:
Can you use `USER_RENAME_LIMIT` and `USER_RENAME_LIMIT_DAYS` for these options?

Generally we avoid nesting options.

(`RATE_LIMIT_PER_NGINX_UPSTREAM` is an outlier because we need potentially add new limits on the fly.)